base/asyncmap.jl

```diff
@@ -22,17 +23,17 @@ end
 
 function AsyncCollector(f, results, c...; ntasks=0)
     if ntasks == 0
-        ntasks = 100
+        ntasks = max(nworkers(), 100)
```

Comment:
i should add that this error occurs with both 0.4.5 and 0.5 daily.  and with `addprocs_sge`, so distributed, not local.